docs/platforms/javascript/guides/aws-lambda/profiling/index.mdx

```diff
@@ -49,14 +49,15 @@ Sentry.init({
   profilesSampleRate: 1.0,
 });
 
+// Profiling happens automatically after setting it up with `Sentry.init()`.
+// However, specific elements can be manually profiled with `startSpan`.
```

Comment:
```suggestion
// Profiling happens automatically after setting it up with `Sentry.init()`.
// Any spans that are captured in your application will have profiling data attached to them
```

maybe like this? 🤔

Reply:
I wanted to explain the `startSpan` part a little bit more. What do you think of this addition?

```md
// Profiling happens automatically after setting it up with `Sentry.init()`. 
// Any spans that are captured in your application will have profiling data attached to them.
// Spans can also be manually captured with `startSpan` like below.
```

Reply:
I like that addition! How's this?

```suggestion
// Profiling happens automatically after setting it up with `Sentry.init()`. 
// All spans captured in your application will have profiling data attached to them.
// You can also manually capture spans with `startSpan`, as shown below:
```

Reply:
sounds very good to me!